AdobeStockImage/Model/SaveLicensedImage.php

```diff
@@ -61,11 +61,15 @@ public function __construct(
     /**
      * @inheritdoc
      */
-    public function execute(int $mediaId, string $destinationPath): void
+    public function execute(int $mediaId, string $destinationPath = null): void
     {
         $imageUrl = $this->client->getImageDownloadUrl($mediaId);
 
         $document = $this->getAssetById->execute($mediaId);
+        $pathAttribute = $document->getCustomAttribute('path');
```

Comment:
Please use the passed destination `path` if path custom attribute does not exist of if's value is null or empty string

Reply:
ok